CHANGELOG.md

```diff
@@ -41,14 +43,18 @@
 - Added support parsing regex flags in REGEX statements for mocked plans. This maintains parity with the `rlike` and `regexp` changes above.
```

Comment:
Could you simplify it to something like this:

Added support for parsing flags in regex statements for mocked plans. This maintains parity with the `rlike` and `regexp` changes above.